nilearn/decoding/space_net.py

```diff
@@ -0,0 +1,1231 @@
+
+    n_alphas : int, optional (default 10).
+        Generate this number of alphas per regularization path.
+        This parameter is mutually exclusive with the `alphas` parameter.
+
+    eps : float, optional (default 1e-3)
+        Length of the path. For example, ``eps=1e-3`` means that
+        ``alpha_min / alpha_max = 1e-3``
+
+    mask : filename, niimg, NiftiMasker instance, optional default None)
+        Mask to be used on data. If an instance of masker is passed,
+        then its mask will be used. If no mask is it will be computed
+        automatically by a MultiNiftiMasker with default parameters.
+
+    target_affine : 3x3 or 4x4 matrix, optional (default None)
+        This parameter is passed to image.resample_img. Please see the
+        related documentation for details.
+
+    target_shape : 3-tuple of integers, optional (default None)
+        This parameter is passed to image.resample_img. Please see the
+        related documentation for details.
+
+    low_pass : False or float, optional, (default None)
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    high_pass : False or float, optional (default None)
+        This parameter is passed to signal. Clean. Please see the related
+        documentation for details
+
+    t_r : float, optional (default None)
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    screening_percentile : float in the interval [0, 100]; Optional (
+    default 20)
+        Percentile value for ANOVA univariate feature selection. A value of
+        100 means 'keep all features'. This percentile is is expressed
+        w.r.t the volume of a standard (MNI152) brain, and so is corrected
+        at runtime to correspond to the volume of the user-supplied mask
+        (which is typically smaller).
+
+    standardize : bool, optional (default True):
+        If set, then we'll center the data (X, y) have mean zero along axis 0.
+        This is here because nearly all linear models will want their data
+        to be centered.
+
+    fit_intercept : bool
+        Fit or not an intercept.
+
+    max_iter : int
+        Defines the iterations for the solver. Defaults to 1000
+
+    tol : float
+        Defines the tolerance for convergence. Defaults to 1e-4.
+
+    verbose : int, optional (default 0)
+        Verbosity level.
+
+    n_jobs : int, optional (default 1)
+        Number of jobs in solving the sub-problems.
+
+    cv : int, a cv generator instance, or None (default 10)
+        The input specifying which cross-validation generator to use.
+        It can be an integer, in which case it is the number of folds in a
+        KFold, None, in which case 3 fold is used, or another object, that
+        will then be used as a cv generator.
+
+    debias: bool, optional (default False)
+        If set, then the estimated weights maps will be debiased.
+
```

Comment:
It is, thx. my comment was: 'int, a cv generator instance, or None (default 10)' could be replaced by
'int, a sklearn.cross_validation.CrossValidation generator instance, or None (default 10)'